PelotonEppSdk/Models/EventCustomField.cs

```diff
@@ -10,18 +11,23 @@ public class EventCustomField
         /// <summary>
         /// The name of this custom field.
         /// </summary>
+        [StringLength(50, MinimumLength = 0, ErrorMessage = "Name must be 50 or fewer characters long.")]
```

Comment:
nit: long is an adverb or adjective,, but you're using it like a noun. Should it read "50 or fewer characters in _length_", or "must contain 50 or fewer _characters_" instead?